### PR TITLE
HW01

### PR DESCRIPTION
**Pull request by Naidantu**
HW01

### CODE REVIEW

**Comment by Naidantu** [comment]
@kaycheharvey
@eburkey

File: hw01/hw01_bfi.Rmd [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+output: 
+  github_document: default
+  html_document: default
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+### Assign bfi data as a new object data
```
Comment (eburkey) [inline hw01/hw01_bfi.Rmd:14]: Great job! I really enjoyed how you organized your work

File: hw01/hw01_bfi.Rmd [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+### Assign bfi data as a new object data 
+
+```{r}
+data <- psych::bfi
+```
+
+### The first part of data
+
+```{r}
```
Comment (eburkey) [inline hw01/hw01_bfi.Rmd:22]: Good job with organizing the data

File: hw01/hw01_bfi.Rmd [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+### Assign bfi data as a new object data 
+
+```{r}
+data <- psych::bfi
```
Comment (eburkey) [inline hw01/hw01_bfi.Rmd:17]: This could use some clarification

**Pull request merged**